docs/docsite/rst/roadmap/ROADMAP_2_12.rst

```diff
@@ -0,0 +1,43 @@
+.. _core_roadmap_2_12:
+
+=================
+Ansible-core 2.12
+=================
+
+.. contents::
+   :local:
+
+Release Schedule
+----------------
+
+Expected
+========
+
+PRs must be raised well in advance of the dates below to have a chance of being included in this ansible-core release.
+
+.. note:: There is no Alpha phase in 2.12.
+.. note:: Dates subject to change.
+
+- 2021-08-30 Feature Freeze
+  No new functionality (including modules/plugins) to any code
+
+- 2021-09-13 Beta 1
+- 2021-09-20 Beta 2 (if necessary)
+
+- 2021-10-04 Release Candidate 1 (and ``stable-2.12`` branching from ``devel``)
+- 2021-10-11 Release Candidate 2 (if necessary)
+
+- 2021-10-25 Release
```

Comment:
only 5 weeks between first pre-release and the release? IMHO, that is too little. I would personally like to see prerelease on pypi appearing at least 2 months before the release.

If there is a feature freeze, why there is no package published for it?

Based on my experience very few people are testing pre-releases and even fewer are testing code from devel branch.

For several Ansible related projects I do have CI/CD pipelines which test prereleases of dependencies, which allow us to find breakages. Still for this to function we need to make the pre-release publishing something common, I would argue that as common as patch-releases on stable branches.